net/net.sh

```diff
@@ -290,7 +299,7 @@ start() {
   echo "Leader deployment took $leaderDeployTime seconds"
   echo "Validator deployment (${#validatorIpList[@]} instances) took $validatorDeployTime seconds"
   echo "Client deployment (${#clientIpList[@]} instances) took $clientDeployTime seconds"
-  echo "Logs in $netLogDir:"
+  echo "Network start logs in $netLogDir:"
```

Comment:
what do you think of putting this stuff (and other like it) in /etc/motd?

Reply:
Yeah I like that idea.  I'd like to not add to this PR though ;)